File: tests/go/cases/case_276_container_handoff_global_07.go
package escape_tests

var retainedCase276 = []map[string]string{}

func Case276ContainerHandoffGlobal07(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "container_handoff_global_07",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "container_handoff_global_07:" + raw,
	}
	box := map[string]map[string]string{"v": payload}
	// ESCAPE: nested container handoff retained globally.
	retainedCase276 = append(retainedCase276, box["v"])
	return "ok"
}

File: tests/nodejs/cases/case_276_container_handoff_global_07.js
/**
 * Task case 276: container_handoff_global_07 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case276ContainerHandoffGlobal07(input) {
  const taskName = 'container_handoff_global_07';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const box = { deep: { value: payload } };
  // ESCAPE: nested indirection retained globally.
  retainedCache.set('nested', box.deep.value);
  return 'ok';
}

module.exports = { case276ContainerHandoffGlobal07 };
